Profiler/log/AttachLog.cpp

```diff
@@ -0,0 +1,29 @@
+#include "AttachLog.h"
+#include "utils/WindowsUtils.h"
+
+
+void AttachLog::createLogFile(std::string path) {
+	FileLogBase::createLogFile(path, "attach.log", false);
+}
+
+
+void AttachLog::logAttach() {
+	char timeStamp[BUFFER_SIZE];
+	getFormattedCurrentTime(timeStamp, sizeof(timeStamp));
+	std::string timeStampString(timeStamp);

```

Comment:
I would extract the logic to get a timestamp string into a helper method to remove duplication in `logDetach()`.